### PR TITLE
Field is required msg wasn't appeared for Entrypoint arguments

### LINKED ISSUE #14675: 'Field is required' msg wasn't appeared for Entrypoint arguments field after cleaning up on the Set up connection page
## Environment
- **Airbyte version**:  0.39.34-alpha
- **OS Version / Instance**: Windows 10
- **Deployment**: OSS
- **Step where error happened**: Setup new connection

## Steps to Reproduce
1. Go to Connections -> Setup source -> Setup destination
2. Scroll down to Normalization & Transformation section on the Setup connection page
3. Click Add transformation
4.  Fill out all required fields
5.  Clean up 'Entrypoint arguments for dbt cli to run the project' field
6.  Click Save transformation

## Current Behavior
Nothing happened. No error message 'Field is required' for 'Entrypoint arguments for dbt cli to run the project' field

## Expected Behavior
'Field is required' error message should appeared for 'Entrypoint arguments for dbt cli to run the project' field

image

### PR DESCRIPTION
**Pull request by m19berger**
## What
Closes [#14675](https://github.com/airbytehq/airbyte/issues/14675)
## How
Fixed field for Entrypoint arguments
## Loom
https://www.loom.com/share/f88bd226282e4cf483b9f1d9da2837b3

### CODE REVIEW

File: airbyte-webapp/src/views/Connection/TransformationForm/TransformationForm.tsx [hunk 1]
```diff
@@ -127,33 +114,40 @@ const TransformationForm: React.FC<TransformationProps> = ({
...
               />
-            </Label>
-          </Column>
+            </ControlLabels>
+          </div>
 
-          <Column>
-            <Label label={<FormattedMessage id="form.transformationType" />}>
+          <div className={styles.column}>
+            <ControlLabels className={styles.label} label={<FormattedMessage id="form.transformationType" />}>
```
Comment (krishnaglick) [inline airbyte-webapp/src/views/Connection/TransformationForm/TransformationForm.tsx:121]: What does this get us? Just curious why the change.

Comment (matter-q) [reply airbyte-webapp/src/views/Connection/TransformationForm/TransformationForm.tsx:121]: Our team decided to do not use styled-components at all and refactor existing

**Review by dizel852 — CHANGES_REQUESTED** [review]
Just tested - "Save transformation" button is enabled if the required field is empty. I guess,  we need to fix it.  

image

**Comment by matter-q** [comment]
> Just tested - "Save transformation" button is enabled if the required field is empty. I guess, we need to fix it.
> 
> image

Nice! Fixed

File: airbyte-webapp/src/views/Connection/TransformationForm/TransformationForm.tsx (hunk 1 above)
Comment (krishnaglick) [reply airbyte-webapp/src/views/Connection/TransformationForm/TransformationForm.tsx:124]: Oh, Label was just a styled component around ControlLabels. Got it!

**Review by dizel852 — APPROVED** [review]
Just checked - fixed)

**Review by krishnaglick — APPROVED** [review]

**Pull request merged**